cdc/owner/changefeed.go

```diff
@@ -47,10 +56,13 @@ type changefeed struct {
 	gcManager        gc.Manager
 	redoManager      redo.LogManager
 
-	schema      *schemaWrap4Owner
+	schema *schemaWrap4Owner
+
 	sink        DDLSink
-	ddlPuller   DDLPuller
-	initialized bool
+	sinkCloseCh chan struct{}
+
+	ddlPuller     DDLPuller
+	runningStatus changeFeedRunningStatus
```

Comment:
Could you add state transfer diagram in comments?

How can we detect violation of state-transfer?